### DISCUSSION TITLE
Play button behavior on Safari

### DISCUSSION THREAD

**Discussion by jordanwmckee** [gilbarbara/react-spotify-web-playback#153] [General]
Not sure if this is a bug or an error on my end, which is why I'm posting it here; but I'm using the player in my website where I load an array of Uris for the player to use. In Chrome, everything works fine, and the player plays the songs when I click the play button once (as expected). However, when I run my site on safari, I have to click the play button twice to get the songs to play (and as of a recent update I believe, the first song in my uri list fails to play regardless of the amount of times I click the play button. However, once I skip the first track, the rest play as expected).

**Comment by RuellePaul** [comment]
Hi,

I'm having exactly the same behavior as described here for Safari (Desktop): have to press two time play button, then it works.

In the console, the first time it does not work, i'm having this message :

```
Unhandled Promise Rejection: NotAllowedError: The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.
```

from this line of code :

```
return this._audioProcessor.resume().then((function() {
    e._player && !e._isPlaying() && Promise.resolve(e._player.play())
 }))
```

Hope this can help !

**Reply by jordanwmckee** [reply]
Thank you! There's also an option on safari for individual websites to enable/disable autoplay, (which I must've missed before) that also solved the issue for me!
